### DISCUSSION TITLE
[x][closed] How to handle replicates?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#109] [Team Posts]
We should specify somewhere what is the intended level of annotation regarding this

**Comment by dzerbino** [comment]
The solution adopted at IHEC is that we can link two samples as being biological replicates of each other.
If two experiments are technical replicates of each other, they are simply attached to the same sample.

**Comment by sveinugu** [comment]
Seems like a good solution. However we have no fields for linking biological samples. Should we add this? Also, how is the state of replicates in the BLUEPRINT data?

**Comment by sveinugu** [comment]
Also, I have been into this in the discussion point https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/6, were I argued for one trackfind sample object for all replicates. However, we should be in line with BioSample IDs, and if I read you correct, that means I should change the example data to have one sample object per replicate (which I don't think it has now). Right?

**Comment by sveinugu** [comment]
Added issue: https://github.com/fairtracks/fairtracks_standard/issues/14. Closing this
